pom.xml

```diff
@@ -57,7 +57,12 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>structs</artifactId>
-      <version>1.2</version>
+      <version>1.7</version>
+    </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>code-coverage-api</artifactId>
+      <version>1.0.0-alpha-2-SNAPSHOT</version>
```

Comment:
In order to get CI running, use `mvn clean package deploy` in https://github.com/jenkinsci/code-coverage-api-plugin/pull/24 . Then you will be able to use this timestamped snapshot here as a dependency